draft-ietf-oauth-browser-based-apps.md

```diff
@@ -207,7 +238,7 @@ the following terms:
 : An application that is dynamically downloaded and executed in a web browser,
   usually written in JavaScript. Also sometimes referred to as a "single-page application", or "SPA".
 
-While this document often refers to "JavaScript applications", this is not intended to be exclusive to the JavaScript language. The recommendations and considerations herein also apply to other languages that execute code in the browser, such as [Web Assembly](https://webassembly.org/).
+This document discusses the security of browser-based applications, which are executed by the browser in a runtime environment. In most scenarios, these applications are JavaScript (JS) applications running in a JavaScript execution environment. Given the popularity of this scenario, this document refers to "JavaScript applications" and to "malicious JavaScript" when discussing attack patterns. Note that the recommendations and considerations in this document are not exclusively linked to the JavaScript language or its runtime. The same considerations apply to other languages and runtimes that allow malicious code to execute in the application's runtime in the browser.
```

Comment:
Was there a reason to remove the mention of Web Assembly?

Reply:
AFAIK, WebAssembly runs in a sandboxed environment (unlike JS). To carry out the attacks we list in this spec, WASM would rely on JS anyway. With this context, it felt unfair to call out WASM here (without much reason), so I opted to reword this and keep it more generic.